src/lib.rs

```diff
@@ -102,10 +102,6 @@
 //! }
 //! ```
 
-#![crate_name = "pnet"]
-#![crate_type = "rlib"]
-#![crate_type = "dylib"]
```

Comment:
Do these have a reason for being here that I missed? My guess was that they were legacy from before you used cargo?

Reply:
The first line was needed pre-cargo.

The other two were to enable building as both static and dynamic libraries... I haven't attempted that in a really long time though, I suspect this isn't needed any more and can be handled automatically by Cargo. Might be worth checking?